src/Jackalope/Transport/TransportInterface.php

```diff
@@ -322,7 +322,7 @@ public function setNodeTypeManager($nodeTypeManager);
      * @return array with the definitions (see
      *      Jackalope\NodeTypeDefinition::fromArray for what is expected)
      *
-     * @see Jackalope\NodeTypeDefinition::fromArray
+     * @see \Jackalope\NodeTypeDefinition::fromArray
```

Comment:
please use a `use` statement for this as well, so here it will be `@see NodeTypeDefinition::fromArray`.